### PR TITLE
update: node.js packages

### PR DESCRIPTION
**Pull request by inureyes**
> [!NOTE]  
> Need to reinstall node packages.
> If you have trouble when running `npm run i`, remove node_modules folder and try again.

**Checklist:** (if applicable)

- [ ] Mention to the original issue
- [ ] Documentation
- [ ] Minium required manager version
- [ ] Specific setting for review (eg., KB link, endpoint or how to setup)
- [ ] Minimum requirements to check during review
- [ ] Test case(s) to demonstrate the difference of before/after

### CODE REVIEW

**Review by agatha197 — APPROVED** [review]
LGTM

File: src/components/backend-ai-webui.ts [hunk 1]
```diff
@@ -11,9 +11,6 @@ import {
   IronFlexFactors,
   IronPositioning,
 } from '../plastics/layout/iron-flex-layout-classes';
-import { Drawer } from '../plastics/mwc/mwc-drawer';
-import '../plastics/mwc/mwc-multi-select';
```
Comment (yomybaby) [inline src/components/backend-ai-webui.ts:15]: @agatha197  "mwc-multi-select" is removed. Is "mwc-multi-select"  not used?

Comment (agatha197) [reply src/components/backend-ai-webui.ts:15]: It is imported but not used.

**Review by yomybaby — APPROVED** [review]
LGTM

**Comment by achimnol** [comment]
It works!

To ensure things working again if you've experience the issue while using `install-dev.sh` in the core mono-repo, please do:
```
cd src/ai/backend/webui
rm -rf node_modules
rm -rf react/node_modules
```

**Pull request merged**